src/spriteloader/grf.cpp

```diff
@@ -276,16 +276,19 @@ uint8_t LoadSpriteV2(SpriteLoader::SpriteCollection &sprite, SpriteFile &file, s
 		/* Type 0xFF indicates either a colourmap or some other non-sprite info; we do not handle them here. */
 		if (type == 0xFF) return 0;
 
-		uint8_t colour = type & SCC_MASK;
+		SpriteComponents colour{type};
+		/* Mask out colour component information from type. */
+		type = type & ~SpriteComponents::MASK;
```

Comment:
```suggestion
		type &= ~SpriteComponents::MASK;
```
nitpick :) And I know, existing code :)